### DISCUSSION TITLE
How is tokenizer's add_prefix_space handled in llama.cpp?

### DISCUSSION THREAD

**Discussion by anhnguyen-namitech** [ggml-org/llama.cpp#6295] [Q&A] [answered]
I'm debugging certain performance issue and I found that HF tokenizer adds prefix space by default. I'm wondering if llama.cpp does the same since it's not obvious in the code.

**Comment by dranger003** [comment, marked as answer]
I think what you are looking for is here?
https://github.com/ggerganov/llama.cpp/blob/ad3a0505e3b6cd777259ee35e61d428357ffc565/llama.cpp#L10912-L10923

**Reply by anhnguyen-namitech** [reply]
Thank you.
